test/ui/RecommendationTest.ts

```diff
@@ -72,6 +75,37 @@ export function RecommendationTest() {
       expect(mainSearchInterface.cmp.componentOptionsModel.getAttributes).toHaveBeenCalled();
     });
 
+    describe('when propaging events', () => {
+      beforeEach(() => {
+        mainSearchInterface.cmp.element.appendChild(test.cmp.element);
+      });
+      it('should prevent model events with atttributes from bubbling', () => {
+        const mainSearchInterfaceContainer = $$('div');
```

Comment:
Could remove some duplication/extract helper function in that particular suite of test

(element creation, spy + event name creation, etc.)